_topics/en/musig.md

```diff
@@ -28,9 +28,19 @@ primary_sources:
     - title: MuSig-DN paper
       link: https://eprint.iacr.org/2020/1057.pdf
 
+    - title: Original MuSig2 implementation (experimental)
+      link: https://github.com/BlockstreamResearch/secp256k1-zkp
+
+    - title: MuSig2 implementation in Libsecp256k1
+      link: https://github.com/bitcoin-core/secp256k1/issues/1479
+
 ## Optional.  Each entry requires "title", "url", and "date".  May also use "feature:
 ## true" to bold entry
 optech_mentions:
+  - title: Libsecp256k1 adds MuSig2 BIP340-compatible multisig as specified in BIP327
```

Comment:
Would you mind moving to the end (most recents)

Reply:
Done (thanks! oops, not sure how I didn't see that, probably am used to ordering by most recent first, as in https://keepachangelog.com/en/1.1.0/)